Dockerfile

```diff
@@ -16,7 +16,7 @@ LABEL maintainer="Rhys Arkins <rhys@arkins.net>" \
   org.opencontainers.image.source="https://github.com/containerbase/buildpack"
 
 #  autoloading buildpack env
-ENV BASH_ENV=/usr/local/etc/env
+ENV BASH_ENV=/usr/local/etc/env PATH=/home/$USER_NAME/bin:$PATH
```

Comment:
We ideally want to avoid using `/home/$USER_NAME` in order to achieve OpenShift compliance but should we refactor that in a different PR?

Reply:
This is the only option to have a fixed path for env without require loading our env file. This should also be compatible with openshift, as the directory will exist, even when openshift will run with a random user.

Our `/home/user/` directory is readable by `root` group

Reply:
Any reason it can't be within `/usr/local` in future?

Reply:
Ok, this would need to make `/usr/local/bin` writable by `root` group, so we can add shell wrapper at runtime.

Maybe allowing use writing to `/usr/local` can be again a showstopper for company usage because of security?

Reply:
Is writing to `/usr/local` unsafe?

Reply:
Not sure, I'm not an expert in those things. I know, that only `root` user is allowed to write there by default.

Reply:
I think you're right, in general it doesn't seem recommended to change /usr/local/bin permissions, e.g. https://unix.stackexchange.com/questions/37724/permissions-ownership-of-usr-local-bin

Reply:
I think OpenShift respects `$HOME` if configured, so we can maybe keep it as `/home/user` regardless of there being a random UID

Reply:
yes, we need to make sure all required folders are writable by root group then.